deformable_ompl/ResamplePathQuery query
---
deformable_ompl/ResamplePathResult result
